types/src/on_chain_config/consensus_config.rs

```diff
@@ -27,15 +27,15 @@ impl ConsensusAlgorithmConfig {
         Self::JolteonV2 {
             main: ConsensusConfigV1::default(),
             quorum_store_enabled: true,
-            order_vote_enabled: false,
+            order_vote_enabled: true,
         }
     }
 
     pub fn default_if_missing() -> Self {
         Self::JolteonV2 {
             main: ConsensusConfigV1::default(),
             quorum_store_enabled: true,
-            order_vote_enabled: false,
+            order_vote_enabled: true,
```

Comment:
I don't think this should be changed, @zekun000 ?

Reply:
we shouldn't change this, it doesn't matter since the config always exist but in principle this should be always off

Reply:
Set the flag to false.